pkg/server/updates.go

```diff
@@ -454,7 +455,20 @@ func (s *Server) reportDiagnostics(ctx context.Context) {
 		return
 	}
 	addInfoToURL(ctx, reportingURL, s, report.Node)
-	res, err := http.Post(reportingURL.String(), "application/x-protobuf", bytes.NewReader(b))
+
+	const timeout = 3 * time.Second
+	client := &http.Client{
```

Comment:
Do we need this one? *this* conn shouldn't fail in weird ways should it?

Reply:
We don't need it for the test, but using a client without a timeout also seemed like a bad idea. Let me know if you have concerns about this and I'll follow up.